scst-store/multicluster-setup.hbs.md

```diff
@@ -149,6 +188,9 @@ Where:
 - `TARGET-REGISTRY-CREDENTIALS-SECRET` is the name of the secret that contains
   the credentials to pull an image from the registry for scanning.
 
+CloudEvent Handler CA CERT `truststore` and `amr-observer-edit-token` are used by observer so that observer can establish a connection with `AMR CloudEvent Handler`
```

Comment:
`by observer` - is this the correct name for it? Is it AMR Observer? Or something else?